Changes

```diff
@@ -88,6 +88,9 @@ Working version
   Alain Frisch and Nathanaëlle Courant)
 
 ### Standard library:
+- #11883, #11884: Update documentation for In_channel and Out_channel
+  with examples and sections to group related functions.
+  (Kiran Gopinathan, review by Daniel Bünzli)
```

Comment:
you should add Xavier Leroy here as well

("review" is not linked to using the github review feature, it basically means "made significant contributions to the PR by giving feedback")

Reply:
Ah, I see, my bad. Got it, updated the changes accordingly.